cmd/ztest/ztest.c

```diff
@@ -6804,14 +6794,25 @@ print_time(hrtime_t t, char *timebuf)
 }
 
 static nvlist_t *
-make_random_props(void)
+make_random_props(ztest_shared_t *zs)
 {
 	nvlist_t *props;
 
-	VERIFY(nvlist_alloc(&props, NV_UNIQUE_NAME, 0) == 0);
+	VERIFY0(nvlist_alloc(&props, NV_UNIQUE_NAME, 0));
 	if (ztest_random(2) == 0)
 		return (props);
-	VERIFY(nvlist_add_uint64(props, "autoreplace", 1) == 0);
+
```

Comment:
Presumably our goal is to _always_ set the failmode property.  In retrospect, I think it would be more clear to add the failmode property to the nvlist either in its own, new, function, or right in-line where ```make_random_props()``` is called.  Otherwise it should be done proper to the earlier ```return (props)``` above.

Reply:
Yes of course, thanks for caching this.